app/routes/internshipProjects/$projectId/index.tsx

```diff
@@ -59,8 +60,7 @@ export default function ProjectDetail() {
 
     try {
       setIsApplying(true);
-
-      await fetcher.submit({}, { method: "put", action: './appliedproject' });
+      await submit({}, { method: "put", action: './appliedproject' });
```

Comment:
why we are not sending data here? 🤔

Reply:
I thought that with the ActionFunction that I have in my "/appliedproject" it was not necessary, but I will add a body with the information that I send.